Chess/src/Board.cpp

```diff
@@ -0,0 +1,290 @@
+#include "Board.h"
+
+/**
+	* Constructs a new Board object and initializes the chessboard with the starting positions of the pieces.
+	*/
+Board::Board()
+{
+	pieces[0][0] = new Rook(WHITE_PLAYER);
```

Comment:
why you're creating the pieces manually what if i want to change the string in the main and test the code ??

Reply:
I have fixed it.